### ISSUE TITLE
highlight affects layout

### ISSUE THREAD

**Issue by extrawurst** [mahulst/bevy_ui_inspector#3]
since the highlighting right now changes elements [border-width](https://github.com/mahulst/bevy_ui_inspector/blob/db860952933940d10060612008a74e18ecf69de1/src/lib.rs#L388C13-L388C25) the very layout we want to debug and style using `bevy_ui_inspector` is affected by the inspector. it would be awesome if we could overlay the highlighting visual without changing the elements itself

**Comment by mahulst** [comment]
This should be fixed in 0.2.1. Not touching the inspected styles anymore. 
I think it should be possible to do it with gizmos, still working on it.

**Comment by mahulst** [comment]
I started a discussion on discord, as I can't get it to work for _any_ UI.
Only when I force every user to opt in with rendering UI to a target camera, like this:
```rust
 commands.spawn((
    NodeBundle {
        background_color: AMBER_500.into(),
        style: Style {
            <style>
            ..default()
        },
        ..default()
    },
    TargetCamera(cam)
));
```
https://discord.com/channels/691052431525675048/1299332466993139815
